### ISSUE TITLE
No Exported Interface/Struct for UpsertOne or CreateOne return value

### ISSUE THREAD

**Issue by FaBeyyy** [steebchen/prisma-client-go#1002]
Given following Example with table name ```Foo```:
```golang
func upsertFoo() 'struct or interface needed' {
    return prismaClient.Foo.UpsertOne(...)
}
```
Is there an **exported** interface or struct that can be used as a return type for the following example?
Should ```db.fooUpsertOne``` be  an exported struct so it can be used as a type?

**Comment by steebchen** [comment]
Fair enough, I'm wondering what's the use case for this? Why are you not doing .Exec(ctx) in that function?

**Comment by FaBeyyy** [comment]
I want to leave it open to the caller if ```Exec``` ```Tx``` etc... is called. If i find some time this week i'll open a PR.
